src/strings.js

```diff
@@ -1,25 +1,27 @@
-function sayHello (string) {
-  // your code here
+cd function sayHello (string) {
+  return 'Hello, ' + string + '!';
+ 
+  
```

Comment:
This looks like a typo, but will prevent your tests from running! Also blank lines should be removed
```suggestion
function sayHello (string) {
  return 'Hello, ' + string + '!';
```